i18n/de/docusaurus-plugin-content-docs/current/help-center/your-account/account-settings.mdx

```diff
@@ -126,7 +126,8 @@ Sie können es Personen aus anderen Organisationen erlauben, Ihr Konto zu finden
 
 ## Passwort ändern
 
-Wenn Sie die Möglichkeit sehen, das Passwort in den Einstellungen zu ändern, können Sie das Passwort ändern. Wenn Sie die Möglichkeit nicht sehen, wird das Passwort in Ihrer Organisation in einem anderen System festgelegt.
+Wenn Sie die Möglichkeit sehen, das Passwort in den Einstellungen zu ändern, können Sie das Passwort ändern. Wenn Sie die Möglichkeit nicht sehen, wird das Passwort in Ihrer Organisation in einem anderen System festgelegt.  
```

Comment:
Wenn die Option für Sie verfügbar ist, können Sie Ihr Passwort in den Einstellungen ändern. Wenn Ihr Account von Ihrer Organisation verwaltet wird, ist diese Option in der Famedly-App möglicherweise nicht verfügbar.